public-ui/src/app/data-browser/views/pm/pm-react.component.tsx

```diff
@@ -0,0 +1,488 @@
+      temp = selectedConcept.analyses.measurementGenderCountAnalysis.filter(
+        a => a.unitName.toLowerCase() ===
+          selectedConceptUnit.toLowerCase());
+      this.setState({selectedConceptValueCountAnalysis: temp[0]});
+    }
+  }
+
+  getValueAnalysis() {
+    const {selectedConceptValueAnalysis, selectedConcept} = this.state;
+    if (!selectedConceptValueAnalysis) {
+      return selectedConcept.analyses.measurementValueGenderAnalysis[0];
+    }
+    return selectedConceptValueAnalysis;
+  }
+
+  getCountAnalysis(conceptUnit: any) {
+    const {selectedConcept, selectedConceptUnit} = this.state;
+    const genderSort = ['Male', 'Female', 'Other'];
+    return selectedConcept.analyses.measurementGenderCountAnalysis.filter(
+      r => r.unitName === selectedConceptUnit)[0].results
+      .sort((a, b) => {
+        return genderSort.indexOf(a.analysisStratumName) -
+          genderSort.indexOf(b.analysisStratumName);
+    });
+  }
+
+  getChartTitle(gender) {
+    return gender.analysisStratumName + ' - ' + (gender.countValue <= 20 ? '&le; ' : '') + gender.countValue.toLocaleString();
+  }
+
+
+  render() {
+    const {loading, selectedGroup, selectedConcept, selectedConceptUnit, unitNames, domainCountAnalysis} = this.state;
+    return <React.Fragment>
+      <style>{styleCss}</style>
+      <div style={styles.pmContainer}>
+        <h1>Browse Program Physical Measurements</h1>
+        { loading ? <Spinner /> :
+        <div className='pm-layout' style={styles.pmLayout}>
+          <aside style={styles.aside}>
+            {
+              PMGroups.map((pmConceptGroup, index) => {
+                const buttonClass = (selectedGroup === pmConceptGroup) ? 'btn btn-link group-button active' : 'btn btn-link group-button';
+                return <div className='button-item' key={index}>
+                  <button className={buttonClass} style={styles.btnLink}
+                  onClick={() => this.showMeasurement(pmConceptGroup, pmConceptGroup.concepts[0])}> {pmConceptGroup.groupName}
+                  </button>
+                  </div>;
+                })
+            }
+          </aside>
+          <div className='db-card' style={styles.dbCard}>
+            <div className='db-card-inner'>
+              <div className='db-card-header'>
+                <div className='group-name'>{selectedGroup.groupName}</div>
+                { selectedConcept && selectedConcept.analyses && selectedConcept.analyses.measurementValueGenderAnalysis ?
+                  <div className='bs-title' style={styles.bsTitle}>
+                    Sex Assigned At Birth
+                    <TooltipReactComponent tooltipKey='pmValueChartHelpText'
+                      label='Physical Measurements tooltip hover' searchTerm='TODO replace search text in here'
+                      action={'Hover on pm biological sex chart of concept' + selectedConcept.conceptName}>
+                    </TooltipReactComponent>
```

Comment:
Use the self-closing syntax:
```suggestion
                      action={'Hover on pm biological sex chart of concept' + selectedConcept.conceptName}/>
```

Reply:
Thanks. changed this.